modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +373,152 @@ ID HOST PORT RACK
+
+=== Multi-AZ optimization
+
+You can configure follower fetching to help ease the latency problems associated with a multi-AZ configuration. This is achieved by configuring consumers to advertise their preferred rack using the `client.rack` option within their consumer configuration. This allows consumers to read data from their closest replica rather than always reading from a (potentially non-local) partition leader. Using this capability has the additional benefit of reducing non-local network traffic and thereby minimizing any cross-AZ network charges.
+
+=== Multi-AZ example
+
+Redpanda provides an official https://github.com/redpanda-data/deployment-automation[deployment automation] project using Ansible and Terraform to help self-hosted users stand up multi-AZ deployments quickly and efficiently.
+
+==== Configure Terraform
+
+First configure the appropriate Terraform script for your cloud provider. Within the deployment-automation project, locate the file for your cloud provider and edit the `availability_zones` parameter. Include each availability zone you intend to use for your deployment. For example, under AWS, edit the `aws/main.tf` file:
+
+[source,bash]
+----
+variable "availability_zone" {
+description = "The AWS AZ to deploy the infrastructure on"
+default     = ["us-west-2a", "us-west-2b", "us-west-2c"]
+type        = list(string)
+}
+----
+
+Alternatively, you can supply the configuration at the command line:
+
+[source,bash]
+----
+$ terraform apply -var=availability_zone='["us-west-2a","us-west-2b","us-west-2c"]'
+----
+
+==== Deploy using Terraform and Ansible
+
+After you configure Terraform for your cloud provider and choose availability zones, you can deploy your cluster. The following example deploys a multi-AZ cluster and validates the rack configuration.
+
+[,bash]
+----
+# Initialize a private key if you haven’t done so already
+ssh-keygen -f ~/.ssh/id_rsa
+
+# Clone the deployment-automation repository
+git clone https://github.com/redpanda-data/deployment-automation
+
+# Choose your cloud provider and initialize Terraform
+pushd deployment-automation/aws # choose one: aws|azure|gcp
+terraform init
+
+# Deploy the infrastructure
+# (Note: This guidance is based on the assumption that you have cloud credentials available)
+terraform apply -var=availability_zone='["us-west-2a","us-west-2b","us-west-2c"]'
+
+# Verify you have correctly specified your racks in the host.ini file:
+popd
+export HOSTS=$(find . -name hosts.ini)
+head -4 $HOSTS
+
+[redpanda]
+34.102.108.41 ansible_user=adminpanda ansible_become=True private_ip=10.168.0.41 rack=us-west2-a
+35.236.32.47 ansible_user=adminpanda ansible_become=True private_ip=10.168.0.39 rack=us-west2-b
+35.236.29.38 ansible_user=adminpanda ansible_become=True private_ip=10.168.0.40 rack=us-west2-c
+
+# Install Ansible Galaxy roles
+ansible-galaxy install -r ./requirements.yml
```

Comment:
With popd / pushd in place, this failed, since I wasn't in the deployment-automation folder. With the above changes in place, this should work.